boards/adafruit-pybadge/Makefile.dep

```diff
@@ -3,6 +3,11 @@ ifneq (,$(filter saul_default,$(USEMODULE)))
 endif
 
 ifneq (,$(filter disp_dev,$(USEMODULE)))
+  USEMODULE += st77xx
```

Comment:
You can just do:

```mk
ifneq (,$(filter disp_dev,$(USEMODULE)))
  USEMODULE += st7735
endif
```
st77xx will be pulled automatically in the `drivers/Makefile.dep`. This is how it works for pseudomodule drivers with Make dependency resolution.